src/impl/source/profiles/profileSync.ts

```diff
@@ -108,15 +109,13 @@ export default class ProfileSync extends ProfileActions {
       for (i = 0, j = profileNames.length; i < j; i += chunk) {
         temparray = profileNames.slice(i, i + chunk);
 
-        var metadataList = await this.profileRetriever.loadProfiles(
-          temparray,
-          this.conn
-        );
+        var metadataList = await this.profileRetriever.loadProfiles(temparray);
 
         let profileWriter = new ProfileWriter();
         for (var count = 0; count < metadataList.length; count++) {
           var profileObj = metadataList[count] as Profile;
-
+          SFPowerkit.log("Reconciling  Tabs", LoggerLevel.DEBUG);
+          await this.reconcileTabs(profileObj);
```

Comment:
I assume that we are reconciling tabs here to remove some noise during retrieve. Shouldn't we do the same for other components types?
Could we reuse the reconcileTab method from the profileReconcile class?

Reply:
@genoud Profile retrieve is working fine for rest of the metadata only for tab we found it is taking extra which failing on deploy again.